### PR TITLE
Add instructions for disabling top-level imports

### PR DESCRIPTION
**Pull request by kant2002**
Based on comment https://github.com/dotnet/runtime/issues/55855#issuecomment-883213093

### CODE REVIEW

**Review by tdykstra — CHANGES_REQUESTED** [review]
Thanks for proposing this change! But I'd prefer to wait until #25250  is merged so we can link to msbuild-props.md#disableimplicitnamespaceimports.

**Comment by RussKie** [comment]
Related https://github.com/dotnet/docs/issues/25436

**Comment by BillWagner** [comment]
ping @tdykstra It looks like #25250 has been merged. Should this be merged now?

File: docs/core/tutorials/top-level-templates.md
Comment (tdykstra) [inline docs/core/tutorials/top-level-templates.md:61]:
```suggestion
If you want to opt-out of that behavior and manually control all namespaces in your project, add [`<DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>`](../project-sdk/msbuild-props.md#disableimplicitnamespaceimports) in the project file.
```

**Comment by tdykstra** [comment]
> Should this be merged now?

Yes, I added the link.

**Review by tdykstra — APPROVED** [review]

**Pull request merged**
